app/src/main/java/com/ucmobiledevelopment/freeways/dto/IncidentDTO.kt

```diff
@@ -1,19 +1,19 @@
 package com.ucmobiledevelopment.freeways.dto
 
 data class IncidentDTO(
-    val CITY: String? = "",
-    val CITY_NAME: String? = "",
-    val COUNTY: String? = "",
-    val COUNTY_NAME: String? = "",
-    val CASE_YEAR: String? = "",
-    val FATALS: String? = "",
-    val LATITUDE: String? = "",
-    val LONGITUDE: String? = "",
-    val STATE: String? = "",
-    val STATE_NAME: String? = "",
-    val ST_CASE: String? = "",
-    val TOTALVEHICLES: String? = "",
-    val TWAY_ID: String? = "",
-    val TWAY_ID2: String? = "",
-    val VE_FORMS: String? = ""
+    internal val CITY: String? = "",
+    internal val CITY_NAME: String? = "",
+    internal val COUNTY: String? = "",
+    internal val COUNTY_NAME: String? = "",
+    internal val CASE_YEAR: String? = "",
+    internal val FATALS: String? = "",
+    internal val LATITUDE: String? = "",
+    internal val LONGITUDE: String? = "",
+    internal val STATE: String? = "",
+    internal val STATE_NAME: String? = "",
+    internal val ST_CASE: String? = "",
+    internal val TOTALVEHICLES: String? = "",
+    internal val TWAY_ID: String? = "",
+    internal val TWAY_ID2: String? = "",
+    internal val VE_FORMS: String? = ""
```

Comment:
I added the internal visibility modifier to implement data hiding. This has had no effect on the program in terms of compiling or stability and it's a great way to keep data safe.

For reference on the internal modifier, here's a snippet directly from kotlinlang.org.
"If you mark it as internal, it will be visible everywhere in the same [module](https://kotlinlang.org/docs/visibility-modifiers.html#modules)."